cmd/ci-operator-prowgen/main.go

```diff
@@ -377,6 +377,9 @@ func generateJobs(
 		if configSpec.PromotionConfiguration != nil {
 			postsubmits[orgrepo] = append(postsubmits[orgrepo], *generatePostsubmitForTest("images", info, true, labels, generatePodSpec(info, "[images]", additionalPostsubmitArgs...)))
 		}
+		if configSpec.PrepublishConfiguration != nil {
```

Comment:
The condition makes the prepublish run only when there's the top-level `prepublish` config, is that intentional? Nothing would be published if you only configured the individual steps...

Reply:
Good point, but I'm also realizing, then, how does anyone ever use rawSteps? Im guessing if you use rawSteps that you might also be required to write the prow job manually? I don't see code in prowgen that handles any kind of raw steps right now.

Reply:
Raw steps get used as intermediates for leaf nodes that do get jobs generated for them